projects/packages/jetpack-mu-wpcom/src/features/starter-page-templates/page-patterns-plugin.tsx

```diff
@@ -43,9 +43,7 @@ export function PagePatternsPlugin( props: PagePatternsPluginProps ) {
 			isWelcomeGuideActive: (
 				select( 'core/edit-post' ) as CoreEditPostPlaceholder
 			 ).isFeatureActive( 'welcomeGuide' ) as boolean, // Gutenberg 7.2.0 or higher
```

Comment:
Does this comment still make sense?
```suggestion
			 ).isFeatureActive( 'welcomeGuide' ) as boolean,
```

Reply:
I considered removing it, but didn't have high confidence as to whether it'd be helpful or not. Likely it's not helpful.